scripts/set-up-contract-tests.sh

```diff
@@ -18,12 +18,14 @@ rm -rf dist/mock_collector*
 rm -rf dist/contract_tests*
 
 # Install python dependency for contract-test
-pip install pymysql
-pip install cryptography
+python3 -m pip install pytest
```

Comment:
adding pytest since i don't have it by default and I think other contributors might face same issue.

the other this is my default python & pip were 2.7 so i'm forcing usage of python3 here.

Reply:
It makes sense to use `python3 -m pip`.